lib/cirneco/doi.rb

```diff
@@ -112,5 +113,29 @@ def check(doi)
         puts "Checksum for #{doi} is not valid"
       end
     end
+
+    desc "transfer DOIs", "transfer list of DOIs"
+    method_option :jwt, :default => ENV['JWT']
```

Comment:
Why not also username and password?

Reply:
So I use JWT because I am calling the LUPO api rather than POODLE (MDS). More comments on that here https://github.com/datacite/cirneco/issues/3 . I thought on adding username and password for that and generate the token, but wouldn't that mean that the gem would need PRIVATE KEY for creating the JSOWEB token? So i thought we couldn't do that. Maybe I am not clear on that.

Reply:
I suggest to support both, as the Lupo API also supports both.